lib/ansible/plugins/inventory/constructed.py

```diff
@@ -45,7 +45,7 @@
         private_only: not (public_dns_name is defined or ip_address is defined)
 
         # complex group membership
-        multi_group: (group_names|intersection(['alpha', 'beta', 'omega']))|length >= 2
+        multi_group: group_names | intersect(['alpha', 'beta', 'omega']) | length >= 2
```

Comment:
Good catch about the filter name! (And I like the extra spacing around `|` :) )

How about keeping the parantheses? I think it makes it easier to understand for the less jinja2-fluent what happens here.
```suggestion
        multi_group: (group_names | intersect(['alpha', 'beta', 'omega'])) | length >= 2
```

Reply:
Sure, either way.